libraries/AP_BoardConfig/px4_drivers.cpp

```diff
@@ -299,7 +299,12 @@ void AP_BoardConfig::px4_setup_drivers(void)
 
     if (px4.board_type == PX4_BOARD_PH2SLIM ||
         px4.board_type == PX4_BOARD_PIXHAWK2) {
-        _imu_target_temperature.set_default(60);
+        _imu_target_temperature.set_default(45);
```

Comment:
ok, so we want it 45C now? Fine. Then why is @proficnc shouting on FB about setting it to 60C still?